client_api.go

```diff
@@ -426,6 +451,42 @@ func (c *Client) ClaimHTTP(ctx context.Context, id, authDomain, scheme string) (
 	return c.claimAssert(ctx, body, id, pathHTTP)
 }
 
+// ClaimEmail requests for an email with a verification link be sent to the
+// provided emailAddress in order for the user to assert control of domain
+// control by following the link inside the sent email. A return value of false
```

Comment:
```suggestion
// by following the link inside the sent email. A return value of false
```
One control is extra no?
.... to assert control of domain control by following....